pkg/utils/args/cluster-identity.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019-2021 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package args
+
+import (
+	"flag"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/validation"
+
+	discoveryv1alpha1 "github.com/liqotech/liqo/apis/discovery/v1alpha1"
+)
+
+// ClusterIdentityFlags stores the values of flags representing a ClusterIdentity.
+type ClusterIdentityFlags struct {
+	ClusterID   *string
+	ClusterName *string
+}
+
+// NewClusterIdentityFlags returns a set of command line flags to read a cluster identity.
+// Example usage:
+//
+//   fcFlags := NewClusterIdentityFlags("foreign-cluster", "The %s of the foreign cluster")
+//   flag.Parse()
+//   foreignClusterIdentity := fcFlags.Read()
+func NewClusterIdentityFlags(prefix, description string) ClusterIdentityFlags {
+	return ClusterIdentityFlags{
+		ClusterID:   flag.String(fmt.Sprintf("%s-id", prefix), "", fmt.Sprintf(description, "ID")),
+		ClusterName: flag.String(fmt.Sprintf("%s-name", prefix), "", fmt.Sprintf(description, "name")),
+	}
+}
```

Comment:
we can define two different functions to create flags for local and for remote clusters, and avoid to pass the description as argument

```go
func NewLocalClusterIdentityFlags
```
and
```go
func NewRemoteClusterIdentityFlags
```